### ISSUE TITLE
0.4~rc16-linvinus1~precise: can't change colors ubuntu 12.04

### ISSUE THREAD

**Issue by 8forty** [linvinus/AltYo#26]
Most configuration changes work fine, but every time I try Application style -> Advanced -> Text color or Background color, when I click the "Apply" button the colors immediately change change back to grey on black on the buttons.  The colors never change on the terminal window.  I've even tried editing the css directly (ayterm-bg-color and ayterm-fg-color), no effect.

Note I get an error when selecting palettes: "AltYo CSS style error, New style will not be saved!, in line 85 at position 20, error:'transition-duration' is not a valid property name".  I removed that line from the CSS and the error went away, but still no color changes.
![image](https://cloud.githubusercontent.com/assets/1214101/7597801/2cf4f9b6-f8b3-11e4-9e00-9d79cad21459.png)

**Comment by linvinus** [comment]
hi! thanks for bug report!
say thanks to GTK3, they are changing gtk-css on every minor release.
i don't know how to support whole range of gtk3 versions.
please remove manually 'transition-duration' property in css text, and don't use theme button after that.
probably, because of that text color didn't saved.

**Comment by 8forty** [comment]
Thanks for the reply, I've heard others complain about GTK3 too, so I know a little about what you're saying.

I did remove the property from the css, still no luck.  I even tried editing the css directly but that doesn't work either.  I changed the line "@define-color ayterm-fg-color #F5F5F5;" thinking it would set the color of the terminal text, is that right?  No matter what I do, the text is still grey and the background still black.

Here's the full css in case that helps:
```
 @define-color ayterm-bg-color alpha(#000000,1.00);
 @define-color ayterm-fg-color #F5F5F5;
 @define-color ayterm-palette-15 #FFFFFF;
 @define-color ayterm-palette-14 #54FFFF;
 @define-color ayterm-palette-13 #FF54FF;
 @define-color ayterm-palette-12 #5454FF;
 @define-color ayterm-palette-11 #FFFF54;
 @define-color ayterm-palette-10 #54FF54;
 @define-color ayterm-palette-9 #FF5454;
 @define-color ayterm-palette-8 #545454;
 @define-color ayterm-palette-7 #AAAAAA;
 @define-color ayterm-palette-6 #00AAAA;
 @define-color ayterm-palette-5 #AA00AA;
 @define-color ayterm-palette-4 #0000AA;
 @define-color ayterm-palette-3 #AA5400;
 @define-color ayterm-palette-2 #00AA00;
 @define-color ayterm-palette-1 #AA0000;
 @define-color ayterm-palette-0 #000000;
 @define-color tab-index-color @ayterm-palette-11;
 @define-color username-color  @ayterm-palette-15;
 @define-color hostname-color  @ayterm-palette-11;
VTToggleButton {
  -VTToggleButton-tab-index-color:@ayterm-palette-11;
  -VTToggleButton-username-color:@ayterm-palette-15;
  -VTToggleButton-hostname-color:@ayterm-palette-11;
  border-color:alpha(#000000,0.0);
  background-color: alpha(#000000,0.0);
  color: #AAAAAA;
  box-shadow: none;
}
VTToggleButton:active{
  background-color: #00AAAA;
  background-image: -gtk-gradient(radial,center center, 0,center center, 1, from (#00BBBB),to (#008888) );
  color: #000000;
}
VTToggleButton:prelight {
  background-color: #AAAAAA;
  background-image: -gtk-gradient(radial,center center, 0,center center, 1, from (#AAAAAA),to (#777777) );
  color: #000000;
}
VTToggleButton:active:prelight{
  background-color: #00AAAA;
  background-image: -gtk-gradient(radial,center center, 0,center center, 1, from (lighter(#00BBBB)),to (#008888) );
  color: #000000;
}
.window_multitabs {
 border-color: #3C3B37;
 border-style: solid;
}
#search_hbox :active {
 border-color: @fg_color;
 color: #FF0000;
}
#search_hbox :prelight {
 background-color: @ayterm-bg-color;
 border-color: @fg_color;
 color: #FF0000;
}
#search_hbox {
 background-color: @ayterm-bg-color;
 border-color: @bg_color;
 color: #00FFAA;
}
HVBox {
 border-color: #3C3B37;
 border-style: solid;
 background-color: @ayterm-bg-color;
}
#OffscreenWindow, VTMainWindow,#HVBox_dnd_window {
 border-style: solid;
 background-color: alpha(@ayterm-bg-color,0.0);
}
HVBox,#quick_options_notebook{
 background-color: @ayterm-bg-color;
}
#settings-scrolledwindow{
 background-color: @bg_color;
}
#settings-scrolledwindow{
 background-color: @bg_color;
}
VTToggleButton{
 box-shadow: none;
}
.window_single_tab {
 border-color: #3C3B37;
 border-style: solid;
}
.window_maximized{
}
.window_multitabs{
}
```

**Comment by linvinus** [comment]
you are right, it was a bug.
in next release will be resolved,
currently you may change it manually in config file in external editor (altyo must be closed)
sorry about that.

**Comment by 8forty** [comment]
Ok.  I did try manually editing ayterm values in the config.ini several times, no luck.

**Comment by linvinus** [comment]
should be resolved in package  altyo - 0.4~rc17-linvinus1~precise

**Comment by Lampwick** [comment]
I still have this same issue in 0.4~rc19
